packages/video_player/video_player/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android, iOS, and web.
 repository: https://github.com/flutter/packages/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.8.3
+version: 2.8.4
 
 environment:
   sdk: ">=3.1.0 <4.0.0"
```

Comment:
These two values need to be rev'd to fix [this failure](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8752915517182459137/+/u/Run_package_tests/analyze_-_legacy/stdout); the package currently fails resolution on 3.13.